### ISSUE TITLE
reportgen wont give output

### ISSUE THREAD

**Issue by Haasie** [soliforte/kismetreportgen#6]
Hello there,

Today i say your nice plugin and thought, i will give it a try.
Unfortunately, i wont give any output. 
I've tried multiple SSID's but none of them work.

Is the plugin not compatible anymore of is it something else ?

with kind regards,

**Comment by dchislop** [comment]
I had a similar problem, it looks like KISMET have released a new 2020 version of their software.

**Comment by jsells02** [comment]
Experienced same issue when trying to use the plugin with Kismet 2020-12-R3. @soliforte have you looked into this yet?

**Comment by ngt500** [comment]
Just to make sure, you hit "Run Report" after entering the SSID and hitting "Add SSID" correct?

**Comment by jsells02** [comment]
@ngt500 yes it fails to produce a report with any content following those steps. There have been some changes to the kismet API from what I was reading so its likely related.

**Comment by ngt500** [comment]
@jsells02 Looks like it has to do with the dot11.device.associated_client_map. I'll try to work on it and submit a pull request

**Comment by ngt500** [comment]
If you go to the url: http://localhost:2501/devices/last-time/0/devices.json while Kismet is running and search for the SSID you have been entering in the report, does it match the field 'kismet.device.base.commonname' for any of the results? I want to make sure there isn't any special characters being missed.

**Comment by BitVenturesUSA** [comment]
Hello, I am experiencing the same issue and I did verify that the SSID I'm trying to run a report for shows up in 'kismet.device.base.commonname' exactly as I'm entering it. Hope this helps!
